src/test/suite/views/webview-app/overview-page.test.tsx

```diff
@@ -12,9 +13,10 @@ const connectionFormTestId = 'connection-form-modal';
 describe('OverviewPage test suite', function () {
   afterEach(() => {
     cleanup();
-    Sinon.restore();
+    sinon.restore();
```

Comment:
nit: I think everywhere in codebase, we stick to sandbox. e.g. `sandbox.restore()`, `sandbox.spy()`.